edge/pkg/edged/edged_status.go

```diff
@@ -261,14 +261,14 @@ func (e *edged) getNodeInfo() (v1.NodeSystemInfo, error) {
 	if err != nil {
 		return nodeInfo, err
 	}
-	nodeInfo.ContainerRuntimeVersion = fmt.Sprintf("remote://%s", runtimeVersion.String())
+	// It's better to call it ContainerRuntimeTypeAndVersion
```

Comment:
Thanks for the fixing, and please delete this comment and squash the commits

Reply:
OK, I'll do that.